### PR TITLE
Basic Home Page Layout Creation

### PR DESCRIPTION
**Pull request by adheeb2**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (ihsanhere) [inline .github/CODEOWNERS:1]: remove adheeb bcz only code owners can access

File: src/pages/index.astro
Comment (ihsanhere) [inline src/pages/index.astro:1]: remove script tag as it is not needed

**Review by ihsanhere — CHANGES_REQUESTED** [review]

**Comment by ihsanhere** [comment]
use swiper for slider
try again with this code as reference

```
---
import 'swiper/css';
---

<div class="swiper">
    <div class="swiper-wrapper">
        <div class="swiper-slide">Slide 1</div>
        <div class="swiper-slide">Slide 2</div>
        <div class="swiper-slide">Slide 3</div>
    </div>
</div>

<script>
  import Swiper from 'swiper';
  const swiper = new Swiper('.swiper', {});
</script>
```

File: src/pages/index.astro
Comment (adheeb2) [reply src/pages/index.astro:1]: it is kept there for the 'know more' button, will change it later

File: .github/CODEOWNERS
Comment (adheeb2) [reply .github/CODEOWNERS:1]: ok

**Comment by adheeb2** [comment]
> use swiper for slider try again with this code as reference
> 
> ```
> ---
> import 'swiper/css';
> ---
> 
> <div class="swiper">
>     <div class="swiper-wrapper">
>         <div class="swiper-slide">Slide 1</div>
>         <div class="swiper-slide">Slide 2</div>
>         <div class="swiper-slide">Slide 3</div>
>     </div>
> </div>
> 
> <script>
>   import Swiper from 'swiper';
>   const swiper = new Swiper('.swiper', {});
> </script>
> ```
have done this type, still does not work

**Review by ihsanhere — APPROVED** [review]

**Review by ihsanhere — CHANGES_REQUESTED** [review]
## Needed Changes

1. ### Change `header` to default style
    remove fixed header style keep the header in only the initial yellow section of home page (same as before)

2. ### Update instagram image
    - get the image by merging from main branch
    - set the instagram image to open in instagram
    - [reference: open instagram from link](https://stackoverflow.com/questions/21295051/link-on-a-webpage-to-open-instagram-app)
    - [reference: making image into link](https://www.geeksforgeeks.org/how-to-turn-an-image-into-a-link-in-html/)
3. ### Create reusable Product Component
    - create a reusable product component in `src/components/Product.astro`
    - [reference: Astro components](https://docs.astro.build/en/basics/astro-components/#component-props)

**Comment by ihsanhere** [comment]
Updated `main` branch with necessary assets
Complete needed changes to close this pull request

**Comment by adheeb2** [comment]
ok noted

**Comment by adheeb2** [comment]
still having problems with the slideshow

**Comment by adheeb2** [comment]
> ## Needed Changes
> 1. ### Change `header` to default style
>    remove fixed header style keep the header in only the initial yellow section of home page (same as before)
> 2. ### Update instagram image
>    
>    * get the image by merging from main branch
>    * set the instagram image to open in instagram
>    * [reference: open instagram from link](https://stackoverflow.com/questions/21295051/link-on-a-webpage-to-open-instagram-app)
>    * [reference: making image into link](https://www.geeksforgeeks.org/how-to-turn-an-image-into-a-link-in-html/)
> 3. ### Create reusable Product Component
>    
>    * create a reusable product component in `src/components/Product.astro`
>    * [reference: Astro components](https://docs.astro.build/en/basics/astro-components/#component-props)

ALL DONE

**Comment by ihsanvp** [comment]
Merging this request for addressing new issues

**Review by ihsanvp — APPROVED** [review]

**Review by ihsanhere — APPROVED** [review]

**Pull request merged**